addon/components/universal-export/file/component.js

```diff
@@ -71,19 +71,19 @@ export default Component.extend({
     'exportLimit.@each.spent',
     'selectedCount',
     function() {
-      let limitPath = `exportLimit.${this.get('selectedType')}.limit`;
-      let spentPath = `exportLimit.${this.get('selectedType')}.spent`;
+      let limitPath = `exportLimit.${this.selectedType}.limit`;
+      let spentPath = `exportLimit.${this.selectedType}.spent`;
 
       if (this.get(limitPath) === -1) {
         return false;
       }
 
-      return (this.get(spentPath) + this.get('selectedCount')) > this.get(limitPath);
+      return (this.get(spentPath) + this.selectedCount) > this.get(limitPath);
```

Comment:
```suggestion
      return (spentPath + this.selectedCount) > this.get(limitPath);
```
Same thing here

Reply:
@JulienVannier66 `spendPath` here is a string based off a condition, `this.get(spentPath)` is to get the right property.